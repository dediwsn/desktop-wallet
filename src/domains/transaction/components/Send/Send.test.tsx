import { render } from "@testing-library/react";
import React from "react";

import { Send } from "./";

describe("Recipient", () => {
	it("should render", () => {
		const { container } = render(<Send />);
		expect(container).toMatchSnapshot();
	});
});
