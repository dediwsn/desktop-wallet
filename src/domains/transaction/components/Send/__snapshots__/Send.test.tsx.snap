// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipient should render 1`] = `
<div>
  <div
    class="sc-fznKkj fPyVlZ"
  >
    <h2>
      Send
    </h2>
    <p
      class="mb-4 text-theme-neutral-600"
    >
      Enter details to send your money
    </p>
    <form
      class="space-y-6"
      data-testid="Form"
      id="send-transaction__form"
    >
      <fieldset
        class="sc-AxhCb ckQmOL relative mt-1 h-20"
      >
        <div
          class="mb-2"
        >
          <label
            class="FormLabel transition-colors duration-100 inline-block text-sm font-semibold text-theme-neutral-dark"
            data-testid="FormLabel"
            for="network"
            label="Network"
          >
            Network
          </label>
        </div>
        <div
          class="sc-fzozJi OgXfh"
        >
          <select
            class="sc-AxgMl cMCunF overflow-hidden shadow-sm w-full bg-theme-background appearance-none rounded border border-theme-neutral-300 py-3 px-4 text-theme-neutral-900 transition-colors duration-200 focus:outline-none focus:border-theme-primary send-transaction__select-network"
            data-testid="Select"
            name="network"
          >
            <option
              disabled=""
              selected=""
              value=""
            >
               
            </option>
            <option
              value="ark"
            >
              Ark Ecosystem
            </option>
          </select>
          <div
            class="sc-AxheI sc-AxmLO ecniej w-12 text-lg pointer-events-none text-theme-neutral-dark"
          >
            <div
              class="sc-AxhUy cPXLjK"
              height="1em"
              width="1em"
            >
              <svg>
                chevron-down.svg
              </svg>
            </div>
          </div>
        </div>
      </fieldset>
      <fieldset
        class="sc-AxhCb ckQmOL relative mt-1 h-20"
      >
        <div
          class="mb-2"
        >
          <label
            class="FormLabel transition-colors duration-100 inline-block text-sm font-semibold text-theme-neutral-dark"
            data-testid="FormLabel"
            for="sender"
            label="Sender"
          >
            Sender
          </label>
        </div>
        <div
          class="sc-fzozJi OgXfh send-transaction__select-contact"
        >
          <div
            class="sc-fzozJi OgXfh"
          >
            <select
              class="sc-AxgMl cMCunF overflow-hidden shadow-sm w-full bg-theme-background appearance-none rounded border border-theme-neutral-300 py-3 px-4 text-theme-neutral-900 transition-colors duration-200 focus:outline-none focus:border-theme-primary pr-12"
              data-testid="Select"
              disabled=""
              name="sender"
            >
              <option
                disabled=""
                selected=""
                value=""
              >
                 
              </option>
              <option
                value="FJKDSALJFKASLJFKSDAJFKFKDSAJFKSAJFKLASJKDFJ"
              >
                FJKDSALJFKASLJFKSDAJFKFKDSAJFKSAJFKLASJKDFJ
              </option>
            </select>
            <div
              class="sc-AxheI sc-AxmLO ecniej w-12 text-lg pointer-events-none text-theme-neutral-dark"
            >
              <div
                class="sc-AxhUy cPXLjK"
                height="1em"
                width="1em"
              >
                <svg>
                  chevron-down.svg
                </svg>
              </div>
            </div>
          </div>
          <div
            class="sc-AxheI sc-AxmLO ecniej"
          >
            <button
              class="px-3 pr-2 text-theme-primary-300 focus:outline-none"
            >
              <div
                class="sc-AxhUy hSYVGy"
                height="20"
                width="20"
              >
                <svg>
                  user.svg
                </svg>
              </div>
            </button>
            <div
              class="sc-AxiKw hGaTKD"
              type="vertical"
            />
            <button
              class="pr-4 pl-2 text-theme-primary-300 focus:outline-none"
            >
              <div
                class="sc-AxhUy hSYVGy"
                height="20"
                width="20"
              >
                <svg>
                  receive.svg
                </svg>
              </div>
            </button>
          </div>
          <div
            class="absolute b-2 flex items-center mt-2 ml-4"
          >
            <div
              class="sc-AxirZ HSPq bg-theme-neutral-200 border-theme-neutral-200"
            />
          </div>
        </div>
      </fieldset>
      <fieldset
        class="sc-AxhCb ckQmOL relative mt-1 h-20"
      >
        <div
          class="mb-2"
        >
          <label
            class="FormLabel transition-colors duration-100 inline-block text-sm font-semibold text-theme-neutral-dark"
            data-testid="FormLabel"
            for="recipient"
            label="Recipient"
          >
            Recipient
          </label>
        </div>
        <div
          class="sc-fzozJi OgXfh send-transaction__select-contact"
        >
          <div
            class="sc-fzozJi OgXfh"
          >
            <select
              class="sc-AxgMl cMCunF overflow-hidden shadow-sm w-full bg-theme-background appearance-none rounded border border-theme-neutral-300 py-3 px-4 text-theme-neutral-900 transition-colors duration-200 focus:outline-none focus:border-theme-primary pr-12"
              data-testid="Select"
              disabled=""
              name="recipient"
            >
              <option
                disabled=""
                selected=""
                value=""
              >
                 
              </option>
              <option
                value="FJKDSALJFKASLJFKSDAJ333FKFKDSAJFKSAJFKLASJKDFJ"
              >
                FJKDSALJFKASLJFKSDAJ333FKFKDSAJFKSAJFKLASJKDFJ
              </option>
              <option
                value="AhFJKDSALJFKASLJFKSDAJ333FKFKDSAJFKSAJFKLASJKDFJ"
              >
                AhFJKDSALJFKASLJFKSDAJ333FKFKDSAJFKSAJFKLASJKDFJ
              </option>
              <option
                value="FAhFJKDSALJFKASLJFKSDAJ333FKFKDSAJFKSAJFKLASJKDFJ"
              >
                FAhFJKDSALJFKASLJFKSDAJ333FKFKDSAJFKSAJFKLASJKDFJ
              </option>
            </select>
            <div
              class="sc-AxheI sc-AxmLO ecniej w-12 text-lg pointer-events-none text-theme-neutral-dark"
            >
              <div
                class="sc-AxhUy cPXLjK"
                height="1em"
                width="1em"
              >
                <svg>
                  chevron-down.svg
                </svg>
              </div>
            </div>
          </div>
          <div
            class="sc-AxheI sc-AxmLO ecniej"
          >
            <button
              class="px-3 pr-2 text-theme-primary-300 focus:outline-none"
            >
              <div
                class="sc-AxhUy hSYVGy"
                height="20"
                width="20"
              >
                <svg>
                  user.svg
                </svg>
              </div>
            </button>
            <div
              class="sc-AxiKw hGaTKD"
              type="vertical"
            />
            <button
              class="pr-4 pl-2 text-theme-primary-300 focus:outline-none"
            >
              <div
                class="sc-AxhUy hSYVGy"
                height="20"
                width="20"
              >
                <svg>
                  receive.svg
                </svg>
              </div>
            </button>
          </div>
          <div
            class="absolute b-2 flex items-center mt-2 ml-4"
          >
            <div
              class="sc-AxirZ HSPq bg-theme-neutral-200 border-theme-neutral-200"
            />
          </div>
        </div>
      </fieldset>
      <fieldset
        class="sc-AxhCb ckQmOL relative mt-1"
      >
        <div
          class="mb-2"
        >
          <label
            class="FormLabel transition-colors duration-100 inline-block text-sm font-semibold text-theme-neutral-dark"
            data-testid="FormLabel"
            for="amount"
            label="Amount ARK"
          >
            Amount ARK
          </label>
        </div>
        <div
          class="sc-fzozJi OgXfh"
        >
          <input
            class="sc-AxgMl cMCunF overflow-hidden shadow-sm w-full bg-theme-background appearance-none rounded border border-theme-neutral-300 py-3 px-4 text-theme-neutral-900 transition-colors duration-200 focus:outline-none focus:border-theme-primary pr-20"
            data-testid="Input"
            disabled=""
            name="amount"
            placeholder="Amount"
            type="number"
          />
          <div
            class="sc-AxheI sc-AxmLO ecniej"
          >
            <button
              class="pr-4 pl-6 text-theme-primary bg-theme-background focus:outline-none bg-theme-neutral-100"
            >
              Send All
            </button>
          </div>
        </div>
      </fieldset>
      <fieldset
        class="sc-AxhCb ckQmOL relative mt-1"
      >
        <div
          class="mb-2"
        >
          <label
            class="FormLabel transition-colors duration-100 inline-block text-sm font-semibold text-theme-neutral-dark"
            data-testid="FormLabel"
            for="smartbridge"
            label="Smartbridge"
          >
            Smartbridge
          </label>
        </div>
        <div
          class="sc-fzozJi OgXfh"
        >
          <input
            class="sc-AxgMl cMCunF overflow-hidden shadow-sm w-full bg-theme-background appearance-none rounded border border-theme-neutral-300 py-3 px-4 text-theme-neutral-900 transition-colors duration-200 focus:outline-none focus:border-theme-primary pr-20"
            data-testid="Input"
            disabled=""
            maxlength="255"
            name="smartbridge"
            placeholder=" "
            type="text"
          />
          <div
            class="sc-AxheI sc-AxmLO ecniej"
          >
            <button
              class="px-4 text-theme-neutral-400 focus:outline-none"
            >
              255 Max
            </button>
          </div>
        </div>
      </fieldset>
      <div
        class="flex"
      >
        <div
          class="w-2/4"
        >
          <fieldset
            class="sc-AxhCb ckQmOL relative mt-1"
          >
            <div
              class="mb-2"
            >
              <label
                class="FormLabel transition-colors duration-100 inline-block text-sm font-semibold text-theme-neutral-dark"
                data-testid="FormLabel"
                for="fee"
                label="Fee ARK"
              >
                Fee ARK
              </label>
              <div
                class="sc-fzozJi OgXfh"
              >
                <input
                  class="sc-AxgMl cMCunF overflow-hidden shadow-sm w-full bg-theme-background appearance-none rounded border border-theme-neutral-300 py-3 px-4 text-theme-neutral-900 transition-colors duration-200 focus:outline-none focus:border-theme-primary "
                  data-testid="Input"
                  name="fee"
                  type="text"
                  value="0"
                />
                <div
                  class="absolute bottom-0 w-full px-1"
                >
                  <div
                    class="flex flex-wrap justify-center"
                    data-testid="Range"
                  >
                    <div
                      class="sc-fzoLsD gcqOdX"
                      data-testid="Range__track"
                      style="transform: scale(1); cursor: pointer;"
                    >
                      <div
                        class="sc-fzpans fDmiZy"
                        data-testid="Range__track__filled"
                      >
                        <div
                          aria-valuemax="100"
                          aria-valuemin="0"
                          aria-valuenow="0"
                          class="sc-fzplWN eSQZBV"
                          data-testid="Range__thumb"
                          draggable="false"
                          role="slider"
                          style="position: absolute; z-index: 0; cursor: grab; user-select: none; transform: translate(0px, 0px);"
                          tabindex="0"
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </fieldset>
        </div>
        <div
          class="w-2/4 mt-7 text-right"
        >
          <div
            class="inline-flex items-center flex-shrink-0 overflow-hidden border rounded border-theme-neutral-300 shadow-sm"
            data-testid="RadioButtonGroup"
            role="radiogroup"
          >
            <button
              aria-checked="false"
              class="sc-fznyAO hCpWvZ relative px-5 py-3 border-transparent transition-colors duration-300 focus:outline-none border-b-3"
              data-testid="RadioButton"
              role="radio"
            >
              <span>
                Last
              </span>
            </button>
            <button
              aria-checked="false"
              class="sc-fznyAO hCpWvZ relative px-5 py-3 border-transparent transition-colors duration-300 focus:outline-none border-b-3"
              data-testid="RadioButton"
              role="radio"
            >
              <span>
                Min
              </span>
            </button>
            <button
              aria-checked="false"
              class="sc-fznyAO hCpWvZ relative px-5 py-3 border-transparent transition-colors duration-300 focus:outline-none border-b-3"
              data-testid="RadioButton"
              role="radio"
            >
              <span>
                Average
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="flex items-center"
      >
        <button
          class="sc-AxjAm MNFIo"
          color="primary"
          disabled=""
          type="button"
        >
          Back
        </button>
        <button
          class="sc-AxjAm jgEKJo ml-5"
          color="primary"
          disabled=""
          type="button"
        >
          Continue
        </button>
      </div>
    </form>
  </div>
</div>
`;
